rasa/shared/core/domain.py

```diff
@@ -714,13 +717,13 @@ def is_retrieval_intent_template(
         """
         return rasa.shared.nlu.constants.RESPONSE_IDENTIFIER_DELIMITER in template[0]
 
-    def setup_slots(self) -> None:
+    def _add_default_slots(self) -> None:
         """Sets up the default slots and slot values for the domain."""
-        self.add_requested_slot()
-        self.add_knowledge_base_slots()
-        self.add_categorical_slot_default_value()
+        self._add_requested_slot()
+        self._add_knowledge_base_slots()
+        self._add_categorical_slot_default_value()
 
-    def add_categorical_slot_default_value(self) -> None:
+    def _add_categorical_slot_default_value(self) -> None:
```

Comment:
@wochinge This part seems to be part of what you fixed in the domain regarding default slots being added while persisting the domain. Do you think the merge is correctly done then?

Reply:
looks good!